frontend/src/pulldasher/index.tsx

```diff
@@ -11,7 +11,7 @@ export const Pulldasher: React.FC = function() {
    const pullsReady = pulls.filter(pull => pull.isReady() && pull.isCiRequired());
    const pullsDevBlocked = pulls.filter(pull => pull.getDevBlock());
    const pullsNeedingCR = pulls.filter(pull => !pull.isCrDone() && !pull.getDevBlock());
-   const pullsNeedingQA = pulls.filter(pull => !pull.isQaDone() && !pull.getDevBlock());
+   const pullsNeedingQA = pulls.filter(pull => !pull.isQaDone() && !pull.getDevBlock() && pull.hasPassedCI());
```

Comment:
I am not sure if I am missing something when looking at how `hasPassedCI` works, but won't it return false if there are no build statuses or CI for the pull?

https://github.com/iFixit/pulldasher/blob/d6fc41cf37a86fd38e69ef8684e10b6dcf18606c/frontend/src/pull.ts#L66-L72

https://github.com/iFixit/pulldasher/blob/d6fc41cf37a86fd38e69ef8684e10b6dcf18606c/frontend/src/pull.ts#L113-L118

Reply:
`[].every(() => false)` is true.  i.e. an empty array will *always* return true for `.every()`

![image](https://user-images.githubusercontent.com/26855/162492098-fb0d7ede-a1a6-43df-8b9e-f2a1ca1ac460.png)

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/every

Reply:
Ooooh sick! Thank you @danielbeardsley